webapp/Dockerfile

```diff
@@ -9,9 +9,10 @@ ENV POSTGRESQL_PASSWORD=vmaas_reader_pwd
 ENV POSTGRESQL_DATABASE=vmaas
 
 RUN yum -y update \
+    && yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
```

Comment:
Just `yum -y install epel-release`

Reply:
Ah! Nice, didn't know it was already available to the image